org.gnome.IconPreview.json

```diff
@@ -0,0 +1,51 @@
+{
+    "app-id" : "org.gnome.IconPreview",
```

Comment:
`org.gnome` is the wrong namespace for this as its not a project under the gnome umbrella.

Reply:
But it kinda is, same problem as Podcasts really

Reply:
If that is the case you just poke the gitlab folks and get it moved under gnome.

Reply:
Should be moved to /GNOME or /World tomorrow